VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -68,7 +84,62 @@ public void deleteProfileImage(@NonNull String username) throws RetroHttpExcepti
         return userService.getBadges(username, page, ApiConstants.STANDARD_PAGE_SIZE);
     }
 
-    public List<EnrolledCoursesResponse> getUserEnrolledCourses(@NonNull String username) throws RetroHttpException {
-        return userService.getUserEnrolledCourses(username);
+    public @NonNull String getUserEnrolledCourses(@NonNull String username) {
+        return config.getApiHostURL() + "/api/mobile/v0.5/users/" + username + "/course_enrollments";
+    }
+
+    public List<EnrolledCoursesResponse> getUserEnrolledCourses(@NonNull String username, boolean tryCache) throws RetroHttpException {
+        List<EnrolledCoursesResponse> ret = null;
```

Comment:
No, it is not a style concern. This method should never return null. It is an implementation error.